openesdh-project-rooms/src/main/java/dk/openesdh/project/rooms/model/CaseSite.java

```diff
@@ -18,12 +19,12 @@
     private String description;
     private SiteVisibility visibility;
     private String nodeRef;
-    private List<SiteMember> siteMembers = new ArrayList<CaseSite.SiteMember>();
-    private List<SiteParty> siteParties = new ArrayList<CaseSite.SiteParty>();
-    private List<CaseDocument> siteDocuments = new ArrayList<>();
+    private List<SiteMember> siteMembers = new ArrayList<>();
+    private List<SiteParty> siteParties = new ArrayList<>();
+    private List<CaseFolderItem> siteDocuments = new ArrayList<>();
```

Comment:
this new ArrayList is not used, I would suggest not to assign it at all, or use Collections.emptyList instead